tests/RepositoryForms/FieldType/Mapper/TextBlockFormMapperTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RepositoryForms\Tests\FieldType\Mapper;
+
+use EzSystems\RepositoryForms\FieldType\Mapper\TextBlockFormMapper;
+
+class TextBlockFormMapperTest extends \PHPUnit_Framework_TestCase
+{
+    private $fieldTypeService;
+
+    protected function setUp()
+    {
+        $this->fieldTypeService = $this->getMockBuilder(\eZ\Publish\API\Repository\FieldTypeService::class)
+            ->getMock();
+        $this->fieldTypeService
+            ->expects($this->any())
+            ->method('getFieldType')
+            ->willReturn($this->getMockBuilder(\eZ\Publish\API\Repository\FieldType::class)->getMock());
+    }
+
+    public function testMapFieldValueFormNoLanguageCode()
+    {
+        $mapper = new TextBlockFormMapper($this->fieldTypeService);
+
+        $config = $this->getMockBuilder(\Symfony\Component\Form\FormConfigInterface::class)->getMock();
+        $config->expects($this->once())
+            ->method('getOption')
+            ->with('languageCode')
+            ->willReturn(false);
+
+        $formFactory = $this->getMockBuilder(\Symfony\Component\Form\FormFactoryInterface::class)
```

Comment:
Why not import those classes ? They're used several times in the test, after all, and I don't think it would reduce readability to do so.